src/styles/blocks/cell.scss

```diff
@@ -0,0 +1,31 @@
+.cell {
+  display: none;
+  width: $cell-size;
+  height: $cell-size;
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
In this task, you don't have other content, but it's better to set the font-family for the entire page (`html` level)